index.html

```diff
@@ -1,8 +1,95 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Memory exercise</title>
-  </head>
-  <body>
-  </body>
+<head>
+  <title>Memory Excercise</title>
+  <style>
+  div#memory_board{
+  	background:#CCC;
+  	border:#999 1px solid;
+  	width:275px;
+  	height:275px;
+  	padding:24px;
+  	margin:0px auto;
+  }
+  div#memory_board > div{
+  	background: url(tile_bg.jpg) no-repeat;
+  	border:#000 1px solid;
+  	width:71px;
+  	height:71px;
+  	float:left;
+  	margin:10px;
+  	padding:20px;
+  	font-size:64px;
+  	cursor:pointer;
+  	text-align:center;
+  }
+  </style>
+  <script>
+  var memory_array = ['A','A','B','B'];
+  var memory_values = [];
+  var memory_tile_ids = [];
+  var tiles_flipped = 0;
+  Array.prototype.memory_tile_shuffle = function(){
```

Comment:
Modifying built-in classes is not a great idea...how else could you achieve this?